front/app/containers/Admin/pagesAndMenu/containers/ProjectsList/ProjectsListContent.tsx

```diff
@@ -83,7 +83,11 @@ const ProjectsListContent = ({ customPage }: Props) => {
               isLastItem={index === adminPublicationsList.length - 1}
               key={adminPublication.id}
             >
-              <ProjectRow publication={adminPublication} actions={['manage']} />
+              <ProjectRow
+                publication={adminPublication}
+                actions={['manage']}
+                hideMoreActions
```

Comment:
Replaced showMoreActions with hideMoreActions. Imo showing something should be the default and hiding it should be the option. Makes it easier to reason about the code.

Reply:
Agreed